### ISSUE TITLE
Error in linking libalpine3d.dylib compiling Alpine3d

### ISSUE THREAD

**Issue by hugocruzz** [EricKeenan/SNOWPACK_WAIS#2]
Good morning,

Compiling Alpine3d and by doing the command make -j18 install, I get the following error (CMAKE_INSTALL_PREFIX = /usr/local) :

[ 56%] Linking CXX shared library ../lib/libalpine3d.dylib
undef: __ZN10SnowpackIO14writeSnowCoverERKN3mio4DateERK11SnowStationRK12ZwischenDataRKm
undef: __ZN11SnowStationC1ERKbS1_S1_
undef: __ZN11ElementDataC1ERKS_
Undefined symbols for architecture x86_64:
  "SnowpackIO::writeSnowCover(mio::Date const&, SnowStation const&, ZwischenData const&, unsigned long const&)", referenced from:
      SnowpackInterface::writeSnowCover(mio::Date const&, std::__1::vector<SnowStation*, std::__1::allocator<SnowStation*> > const&) in lto.o
  "SnowStation::SnowStation(bool const&, bool const&, bool const&)", referenced from:
      SnowpackInterface::readInitalSnowCover(std::__1::vector<SnowStation*, std::__1::allocator<SnowStation*> >&, std::__1::vector<std::__1::pair<unsigned long, unsigned long>, std::__1::allocator<std::__1::pair<unsigned long, unsigned long> > >&) in lto.o
  "ElementData::ElementData(ElementData const&)", referenced from:
      std::__1::vector<ElementData, std::__1::allocator<ElementData> >::vector(std::__1::vector<ElementData, std::__1::allocator<ElementData> > const&) in lto.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [lib/libalpine3d.3.2.1.dylib] Error 1
make[1]: *** [alpine3d/CMakeFiles/alpine3d.dir/all] Error 2
make: *** [all] Error 2

After cmake . I get : 
$ cmake .
-- Could NOT find Subversion (missing: Subversion_SVN_EXECUTABLE) 
-- Found METEOIO 
-- Found LIBSNOWPACK 
-- Could NOT find Doxygen (missing: DOXYGEN_EXECUTABLE) 
-- Could NOT find Doxygen (missing: DOXYGEN_EXECUTABLE) 
-- Configuring done
-- Generating done
-- Build files have been written to: <path>/SNOWPACK_WAIS/snowpack/Source/alpine3d

Do you have any idea of where the problem could come from or what I did wrong ?

**Comment by EricKeenan** [comment]
Did you compile `meteoio` and `SNOWPACK`?

**Comment by hugocruzz** [comment]
Yes, i'm thinking that the problem comes from my C compiler. I'm running it on OSX. I will just try on linux or windows and I think it should work.
Cheers

**Comment by EricKeenan** [comment]
Ok, great! Good luck. Don't hesitate to reach out if you have any questions.
